webui/src/components/BranchesPage.js

```diff
@@ -131,6 +156,16 @@ const BranchesPage = ({repo, branches, listBranches, listBranchesPaginate, creat
                                 </div>
                                 <div className="float-right">
                                     <ButtonGroup className="branch-actions">
+                                        {(repo.default_branch !== branch) ? 
```

Comment:
Please add a description when the mouse hover on the button (see 'tooltip' from other buttons):
"delete the branch"